bridges/snowbridge/primitives/core/src/rewards.rs

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023 Snowfork <hello@snowfork.com>
+
+use frame_support::pallet_prelude::DispatchResult;
+
+pub trait RewardLedger<AccountId, Balance> {
+    // Deposit reward which can later be claimed by `account`
+    fn deposit(account: AccountId, value: Balance) -> DispatchResult;
```

Comment:
@claravanstaden can you please point me to the place in Snowbridge pallets where you want to call this?